declarations/Scalingo.history.json

```
@@ -0,0 +1,22 @@
+{
+  "Terms of Service": [
+    {
+      "fetch": "https://scalingo.com/terms-of-service",
+      "select": [
+        ".title",
+        ".mx-sm-auto"
+      ],
+      "validUntil": "2021-08-16T14:36:00.000Z"
+    }
+  ],
+  "Data Processor Agreement": [
+    {
+      "fetch": "https://scalingo.com/data-processing-agreement",
+      "select": [
+        ".title",
+        ".mx-sm-auto"
+      ],
+      "validUntil": "2021-08-16T14:36:00.000Z"
```

Isn't this November? 🤔

https://github.com/OpenTermsArchive/contrib-declarations/issues/46

```suggestion
      "validUntil": "2021-11-16T14:36:00.000Z"
```